web-common/src/components/data-graphic/marks/segment.ts

```diff
@@ -1,39 +1,50 @@
+function pathIsDefined(yAccessor: string) {
+  return (d: Record<string, number>) => {
+    const val = d[yAccessor];
+    return !(
+      val === undefined ||
+      (typeof val === "number" && isNaN(val)) ||
+      val === null
+    );
+  };
+}
+
 /**
  * Helper function to compute the contiguous segments of the data
  * based on https://github.com/pbeshai/d3-line-chunked/blob/master/src/lineChunked.js
  */
-export function computeSegments(lineData, defined, isNext = () => true) {
+export function computeSegments(
```

Comment:
Note that `computeSegments` is only ever called in one place, and the optional argument `isNext = () => true` was not given. `isNext` was only used in the chunk
```
      if (isNext(lastDatum, d)) {
        ...
      } else {
       ...
     }
```

Since `isNext` always returns true, the if block is always used and the else block is never used.

Additionally, `lastDatum` is only used as an argument to `isNext`, so it's not need either.


Finally, the `defined` arg to this fn is `pathIsDefined(yAccessor)` at the only call site, and `pathIsDefined` is never used anywhere other than at that call site, so pulling that into this file and inlining it in this fn